config/testgrids/kubevirt/gen-config.yaml

```diff
@@ -0,0 +1,96 @@
+dashboard_groups:
+- dashboard_names:
+  - kubevirt-presubmits
+  name: kubevirt
+dashboards:
+- dashboard_tab:
+  - code_search_url_template:
+      url: https://github.com/kubevirt/project-infra/compare/<start-custom-0>...<end-custom-0>
+    name: pull-kubevirt-e2e-k8s-1.20
+    open_bug_template:
+      url: https://github.com/kubevirt/project-infra/issues/
+    test_group_name: pull-kubevirt-e2e-k8s-1.20
+  - code_search_url_template:
```

Comment:
I think here and below is a section missing for `pull-kubevirt-e2e-k8s-1.18`

Reply:
good catch, thx! should be fixed now